guides/source/active_job_basics.md

```diff
@@ -131,77 +119,274 @@ That's it!
+| **concurrency_maintenance**          | Whether the dispatcher performs concurrency maintenance work.                                       | true                                          |
+
+You can read more about these [configuration options in the Solid Queue documentation](https://github.com/rails/solid_queue?tab=readme-ov-file#configuration). There are also [additional configuration options](https://github.com/rails/solid_queue?tab=readme-ov-file#other-configuration-settings) that can be set in `config/<environment>.rb` to further configure Solid Queue in your Rails Application.
+
+### Queue Order
+
+As per the configuration options in the [Configuration section](#configuration),
+the `queues` configuration option will list the queues that workers will pick
+jobs from. In a list of queues, the order matters. Workers will pick jobs from
+the first queue in the list - once there are no more jobs in the first queue,
+only then will it move onto the second, and so on.
+
+```yaml
+# config/queue.yml
+production:
+  workers:
+    - queues:[active_storage*, mailers]
+      threads: 3
+      polling_interval: 5
+```
+
+In the above example, workers will fetch jobs from queues starting with "active_storage", like  the `active_storage_analyse` queue and `active_storage_transform` queue. Only when no jobs remain in the `active_storage`-prefixed queues will workers move on to the `mailers` queue.
+
+NOTE: The wildcard `*` (at the end of "production") is only allowed on its own or at the end of a queue name to match all queues with the same prefix. You can't specify queue names such as `*_some_queue`.
+
+WARNING: Using wildcard queue names (e.g., `queues: active_storage*`) can slow down polling performance due to the need for a `DISTINCT` query to identify all matching queues, which can be slow on large tables. For better performance, it’s best to specify exact queue names instead of using wildcards.
+
+Active Job supports positive integer priorities when enqueuing jobs. You can read more about the [Priority section](#priority). This setup is helpful when you have jobs with different levels of importance or urgency in the same queue. Within a single queue, jobs are picked based on their priority (with lower values being higher priority). However, when you have multiple queues, the order of the queues themselves takes priority.
+
+For example, if you have two queues, `production` and `background`, jobs in the `production` queue will always be processed first, even if some jobs in the `background` queue have a higher priority.
+
+WARNING: You should avoid using the `priority` option if you're relying on the
+order of the queues list. If both the order of the queues and the priority
+option are used, the queue order will take precedence, and the priority option
+will only apply within each queue.
```

Comment:
This is already explained above, with an example. This can be removed I think? Feels repetitive.

Reply:
✅